todo-frontend/src/components/Edit.vue

```diff
@@ -1,62 +1,44 @@
 <template>
     <div id="editForm">
-        <p>Title: <input v-model="forEdit.title" type="text"></p>
-        <p>Description: <input v-model="forEdit.description" type="text"></p>
-        <p>Priority: 
-            <select v-model="forEdit.priority">
-                <option value="LOW">LOW</option>
-                <option value="MEDIUM">MEDIUM</option>
-                <option value="HIGH">HIGH</option>
-            </select>
-        </p>
-
-        <p>Done: <input v-model="forEdit.done" type="checkbox"></p>
-
-
-        <button @click="submit()">Submit</button>
+        <app-todo-form :propTitle="forEdit.title" :propDescription="forEdit.description" 
+        :propPriority="forEdit.priority" @formSubmit="updateNew($event)"></app-todo-form>
```

Comment:
@formSubmit="updateNew($event)
trebalo bi da moze i bez ($event) posto to po defaultu prosledjuje